src/Gui/Dialogs/DlgAbout.cpp

```diff
@@ -652,6 +653,28 @@ void AboutDialog::copyToClipboard()
     str << "Qt " << QT_VERSION_STR << ", ";
     str << "Coin " << COIN_VERSION << ", ";
     str << "Vtk " << fcVtkVersion << ", ";
+
+    const char* cmd = "import ifcopenshell\n"
+                      "version = ifcopenshell.version";
+    PyObject * ifcopenshellVer = nullptr;
+
+    try {
+        ifcopenshellVer = Base::Interpreter().getValue(cmd, "version");
+    }
+    catch (const Base::Exception& e) {
+        Base::Console().Debug("%s (safe to ignore, unless using the BIM workbench and IFC).\n", e.what());
```

Comment:
```suggestion
        Base::Console().DeveloperWarning("%s (safe to ignore, unless using the BIM workbench and IFC).\n", e.what());
```

It turns out there wasn't a `Debug` type of message. I believe `DeveloperWarning` is the next equivalent.